test/test_models.py

```diff
@@ -29,7 +29,7 @@ def list_model_fns(module):
     return [get_model_builder(name) for name in list_models(module)]
 
 
-def _get_image(input_shape, real_image, device):
+def _get_image(input_shape, real_image, device, weights=None, dtype=None):
```

Comment:
Below in the docs:

> Currently, the real image is utilized for the following list of models:

We probably want to remove that list because it will be out of date quite often

Reply:
Make sense, will remove this comment!